docs/index.rst

```diff
@@ -7,7 +7,7 @@ Rally |release|
     .. warning::
 
         This documentation is for the version of Rally currently under development.
-        Were you looking for the `documentation of the latest stable version <//esrally.readthedocs.io/en/stable/>`_?
+        Were you looking for the `documentation of the latest stable version <https://esrally.readthedocs.io/en/stable/>`_?
```

Comment:
In case you were wondering, this is a protocol-relative URL that was useful when Read the Docs wasn't enforcing HTTPS.